src/main/java/co/refiere/resources/util/RefiereInterceptor.java

```diff
@@ -1,108 +1,140 @@
 package co.refiere.resources.util;
 
+import java.io.File;
+import java.io.IOException;
 import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.List;
 
 import javax.mail.MessagingException;
-import javax.mail.internet.AddressException;
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.apache.commons.io.FileUtils;
 import org.hibernate.CallbackException;
 import org.hibernate.EmptyInterceptor;
 import org.hibernate.ScrollMode;
 import org.hibernate.ScrollableResults;
 import org.hibernate.StatelessSession;
 import org.hibernate.type.Type;
 
+
 import co.refiere.dao.PersonDao;
 import co.refiere.models.Campaign;
 import co.refiere.models.Company;
 import co.refiere.models.Person;
+import co.refiere.models.ReferencesCodes;
 import co.refiere.resources.base.EmailRequest;
 import co.refiere.services.mailer.RefiereServiceFactory;
 
 public class RefiereInterceptor extends EmptyInterceptor {
 
-    /**
-     * onSave – Called when you save an object, the object is not save into database yet.
-     * onFlushDirty – Called when you update an object, the object is not update into database yet.
-     * onDelete – Called when you delete an object, the object is not delete into database yet.
-     * preFlush – Called before the saved, updated or deleted objects are committed to database (usually before postFlush).
-     * postFlush – Called after the saved, updated or deleted objects are committed to database.
-     */
-    private static final long serialVersionUID = 1L;
-    private static final Log log = LogFactory.getLog(RefiereInterceptor.class);
+	/**
+	 * onSave – Called when you save an object, the object is not save into database yet.
+	 * onFlushDirty – Called when you update an object, the object is not update into database yet.
+	 * onDelete – Called when you delete an object, the object is not delete into database yet.
+	 * preFlush – Called before the saved, updated or deleted objects are committed to database (usually before postFlush).
+	 * postFlush – Called after the saved, updated or deleted objects are committed to database.
+	 */
+	private static final long serialVersionUID = 1L;
+	private static final Log log = LogFactory.getLog(RefiereInterceptor.class);
 
-    public RefiereInterceptor() {
-    }
+	public RefiereInterceptor() {
+	}
 
-    public boolean onSave(Object entity,Serializable id,
-            Object[] state,String[] propertyNames,Type[] types)
-                    throws CallbackException {
+	public String getStringfontTemplate(String fileName){
```

Comment:
There is a ResourceManager class which do something similar like this, please check it and use it if apply.